docs/src/appendix/cast/multicall.md

```diff
@@ -27,4 +34,18 @@ call_type = "invoke"
 contract_address = "map_contract"
 function = "put"
 inputs = ["0x123", "234"]
-```
\ No newline at end of file
+```
+
+## `new`
+
+Generates an empty template for the multicall `.toml` file that may be later use with the `run` subcommand.
+
+### `--output_path, -p <PATH>`
+Optional.
+
+When provided, it specifies where the template should be saved. If omitted, the template contents are going to be printed out to the stdout.
```

Comment:
```suggestion
Specifies the path where the template will be saved.
```

btw, does this specify path to the directory or the file?

Reply:
What's wrong with the sentence that is there already? 

> btw, does this specify path to the directory or the file?

it has to be file

Reply:
it's unnecessarily long. Docs should be concise, but informative.

> it has to be file
cool, then let's state it explicitly, that this is a path to the file to which template will be saved.

Reply:
> it's unnecessarily long. Docs should be concise, but informative.

Ok, but I do not think the line you suggested is good. This is because it is missing what is said in the original one (what happens when you provide this arg and what happens if you do not).

I also don't think the original line is botheringly long.

Reply:
> cool, then let's state it explicitly, that this is a path to the file to which template will be saved.

[dox fdbck](https://github.com/foundry-rs/starknet-foundry/pull/98/commits/3a6034e60aabceb786be6e99c41badae1fb8543a)

Reply:
[updt](https://github.com/foundry-rs/starknet-foundry/pull/98/commits/222bb30e6236945f904fd79a0cc66021efc637f2)